lib/matplotlib/axes/_base.py

```diff
@@ -3342,15 +3346,48 @@ def _set_view_from_bbox(self, bbox, direction='in',
         twiny : bool
             Whether this axis is twinned in the *y*-direction.
         """
+        Xmin, Xmax = self.get_xlim()
+        Ymin, Ymax = self.get_ylim()
+
+        if(len(bbox) == 3):
+            # Zooming code
+            xp, yp, scl = bbox
+            print("in set view, for zooming with scale {}".format(scl))
+
+            # Should not happen
+            if(scl == 0):
+                scl = 1.
```

Comment:
yes, good idea. I'll just print a warning and return. Since it's harmless, I think there's no need to raise any exceptions. However, in that case, the user will see quite a few print statements if they repeat the call with wrong (or even just void) arguments. I'm still new to python (been 4 months, coming from yorick, matlab like open source lang and C), so I'm not completely familiar with the pythonic way of error and warning reporting. Open to recommendations.